### PR TITLE
feat(newserver): Colony Survival

### LINKED ISSUE #1974: [Server Request] Colony Survival
**What game server Would you like adding?**
Colony Survival

**Can you link to any tutorials or guides?**
https://colonysurvival.gamepedia.com/Dedicated_Server#Installation_.28Linux.29

**Is the server on Steam?**
748090

**Is the server supported on Linux?**
Yes with mono.

### PR DESCRIPTION
**Pull request by h3o66**
# Description

Adds new Game Colony Survival

Fixes #1974

## Type of change

* [ ] Bug fix (a change which fixes an issue).
* [ ] New feature (change which adds functionality).
* [x] New Server (new server added).
* [ ] Refactor (restructures existing code).
* [ ] Comment update (typo, spelling, explanation, examples, etc).

## Checklist

PR will not be merged until all steps are complete.

* [x] This pull request links to an issue.
* [x] This pull request uses the `develop` branch as its base.
* [x] This pull request Subject follows the Conventional Commits standard.
* [x] This code follows the style guidelines of this project.
* [x] I have performed a self-review of my code.
* [x] I have checked that this code is commented where required.
* [x] I have provided a detailed with enough description of this PR.
* [x] I have checked If documentation needs updating.

## Documentation

If documentation does need updating either update it by creating a PR (preferred) or request a documentation update.
* User docs: https://github.com/GameServerManagers/LinuxGSM-Docs
* Dev docs: https://github.com/GameServerManagers/LinuxGSM-Dev-Docs

**Thank you for your Pull Request!**

### CODE REVIEW

File: lgsm/functions/info_messages.sh [hunk 1]
```diff
@@ -1446,6 +1446,16 @@ fn_info_message_warfork(){
 	} | column -s $'\t' -t
 }
 
+fn_info_message_colony(){
+	echo -e "netstat -atunp | grep colonyserver"
+        echo -e ""
```
Comment (dgibbs64) [inline lgsm/functions/info_messages.sh:1451]: using spaces instead of tabs

**Comment by dgibbs64** [comment]
I tested and Im getting this error

```
[2020-11-22_19-25-00][Warning]:Stopped due to missing options or serverlock fail
```

any ideas?

**Comment by h3o66** [comment]
New changes with the latest update 0.8.1 :
- Generate a random seed if none is provided via the JSON-config launch option
- Automatically fall back to loading a world if the world passed via JSON-config launch exists

With this no more fix is required from our side.

**Comment by h3o66** [comment]
The new changes look good to me, if someone else wants to test feel free to do so.

**Review by dgibbs64 — CHANGES_REQUESTED** [review]
Please get monitor working correctly. Requires a query port for port querying (if supported)
```
./colserver query-raw
```
command might be useful doe tou
```
lgsm@lgsm-ubuntu-2004:~/colserver$ ./colserver m
[  OK  ] Monitoring colserver: Checking session: OK
[ ERROR ] Monitoring colserver: Checking port: Unable to query, queryport is not setERROR
```

**Comment by h3o66** [comment]
> Please get monitor working correctly. Requires a query port for port querying (if supported)
> 
> ```
> ./colserver query-raw
> ```
> 
> command might be useful doe tou
> 
> ```
> lgsm@lgsm-ubuntu-2004:~/colserver$ ./colserver m
> [  OK  ] Monitoring colserver: Checking session: OK
> [ ERROR ] Monitoring colserver: Checking port: Unable to query, queryport is not setERROR
> ```

should be working now with the last commit.

**Review by dgibbs64 — APPROVED** [review]

**Pull request merged**